### PR TITLE
Add background info

### PR DESCRIPTION
**Pull request by treykris**
Links
-----
- Server repo PR: https://github.com/UPchieve/server/pull/401

Description
-----------
- Added background information action to the approval steps card 
- Added background information view
- Account Action items are sorted by completed / pending / default
- Test for testing user submits background info

TODO:
- [ ] Finalize copy / question options

Developer self-review checklist
-------------------------------
- [ ] Potentially confusing code has been explained with comments
- [ ] No warnings or errors have been introduced; all known error cases have been handled
- [ ] Any appropriate documentation (within the code, README.md, docs, etc) has been updated
- [ ] All edge cases have been addressed

### CODE REVIEW

File: src/views/DashboardView/VolunteerDashboard/index.vue [hunk 1]
```diff
@@ -369,6 +403,13 @@ export default {
           value: `${numElapsedAvailabilityHours} hours elapsed`
         }
       ];
+    },
+
+    approvalCardSubheader() {
+      if (this.user.volunteerPartnerOrg)
+        return "Provide background information to become an approved volunteer";
```
Comment (treykris) [inline src/views/DashboardView/VolunteerDashboard/index.vue:410]: TODO: Copy

File: src/views/BackgroundInfoView.vue [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+          <li class="uc-form-col">
+            <label for="linkedin" class="uc-form-label">
+              If you have one, please provide us with a link to your LinkedIn
+              profile.
+            </label>
+            <p class="background-info__question-description">(optional)</p>
+
+            <input
+              type="text"
+              pattern=".*linkedin\.com.*\/in\/.*"
```
Comment (jkeat) [inline src/views/BackgroundInfoView.vue:91]: With the pattern attribute, you can use the `:invalid` pseudo selector for changing the input's border on non-matches
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern#Examples

Comment (treykris) [reply src/views/BackgroundInfoView.vue:91]: This is awesome! thank you

**Comment by jkeat** [comment]
@treykris still waiting on copy to be finalized for this?

**Comment by treykris** [comment]
@jkeat I believe so

**Pull request merged**